package.json

```diff
@@ -101,11 +101,11 @@
     "start:css": "npm run bundle:css && npm run bundle:css -- -w",
     "start:html": "npm run bundle:html -- --baseUrl http://localhost:1963/",
     "start:server": "live-server data/content --port=1963 --no-browser",
-    "firefox": "jpm run -b beta",
+    "firefox": "jpm run -b beta --prefs ./dev-prefs.json",
```

Comment:
you might need to change yamscripts.yml and run `npm run yamscripts` to regenerate the package.json for the tasks